docs/user/site.rst

```diff
@@ -229,6 +229,16 @@ mesh_vpn
                   -- You can also omit the ipv4 to allow both connection via ipv4 and ipv6
                   remotes = {'"vpn2.alpha-centauri.freifunk.net" port 10000'},
                 },
+                peer3 = {
+                  key = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
+                  -- In addition to domains you can also add ip addresses, which provides
+                  -- resilience in case of dns outages
+                  remotes = {
+                    '"vpn3.alpha-centauri.freifunk.net" port 10000',
+                    '2001:db8::3:1 port 10000',
```

Comment:
Ah, I forgot to mention that the IPv6 address needs to be enclosed in `[ ]` like this (and the port could be added with a colon instead of using the "port" keyword, but that doesn't really matter).